jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/AbstractConnection.java

```diff
@@ -75,7 +75,7 @@ public void setInputBufferSize(int inputBufferSize)
         _inputBufferSize = inputBufferSize;
     }
 
-    protected Executor getExecutor()
+    public Executor getExecutor()
```

Comment:
I would leave this method `protected` and override it in `HTTP2Connection` to change the visibility to public, like we do in `WebSocketConnection`.